.ctags

```diff
@@ -0,0 +1,13 @@
+--langdef=Elixir
```

Comment:
Because someone maybe would not use them, however I have no strong feelings about this. i.e. -> it's fine